src/types/kubectl.ts

```diff
@@ -70,13 +70,20 @@ export class Kubectl {
       let newReplicaSet = ''
       if (result?.stdout) {
          const stdout = result.stdout.split('\n')
+         core.debug('stdout from getNewReplicaSet is ' + JSON.stringify(stdout))
          stdout.forEach((line: string) => {
             const newreplicaset = 'newreplicaset'
             if (line && line.toLowerCase().indexOf(newreplicaset) > -1)
```

Comment:
Something seems off here. The if statement is just being used to log a single debug statement?

Reply:
newReplicaSet = below will not work correctly unless it's inside this if statement

Reply:
good catch!